<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Musli Player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QTableWidget" name="playlist">
        <property name="cursor" stdset="0">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="rowCount">
         <number>0</number>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <column>
         <property name="text">
          <string>Название</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Длительность</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="lengthBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="trackSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <widget class="QToolButton" name="playBtn">
          <property name="text">
           <string>Играть</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="popupMode">
           <enum>QToolButton::DelayedPopup</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="previousBtn">
          <property name="text">
           <string>Предыдущий</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="pauseBtn">
          <property name="text">
           <string>Пауза</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="nextBtn">
          <property name="text">
           <string>Следующий</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="stopBtn">
          <property name="text">
           <string>Стоп</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDial" name="volumeDial">
          <property name="toolTip">
           <string>100%</string>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="invertedControls">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>449</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Медиа</string>
    </property>
    <addaction name="openFile"/>
    <addaction name="openUrl"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Плейлист</string>
    </property>
    <addaction name="openPlaylist"/>
    <addaction name="clearPlaylist"/>
   </widget>
   <widget class="QMenu" name="radioMenu">
    <property name="title">
     <string>Радио</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="radioMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openFile">
   <property name="text">
    <string>Открыть файл</string>
   </property>
  </action>
  <action name="openUrl">
   <property name="text">
    <string>Открыть ссылку</string>
   </property>
  </action>
  <action name="openPlaylist">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="clearPlaylist">
   <property name="text">
    <string>Очистить</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
